apps/api-gateway/src/issuance/issuance.service.ts

```diff
@@ -129,9 +129,9 @@ export class IssuanceService extends BaseService {
         return this.sendNatsMessage(this.issuanceProxy, 'retry-bulk-credentials', payload);
     }
 
-    async _getWebhookUrl(tenantId: string): Promise<string> {
+    async _getWebhookUrl(tenantId?: string, orgId?: string): Promise<string> {
```

Comment:
`tenantId`,`orgId` both are optional parameter to get webhook URL ?

Reply:
Yes, for shared agent, we need tenantId for fetching the webhook Url details and for dedicated agent we need OrgId for fetching the details, That's why I have made both the arguments are optional